### PR TITLE
Fix development

### PR DESCRIPTION
**Pull request by JeffreyDevloo**
- Fixes edge reroute test (enables its pushing to testrail + Unknown reference error)
- Fixes vpool test for ee and ose versions

### CODE REVIEW

**Comment by JeffreyDevloo** [comment]
Depends on https://github.com/openvstorage/integrationtests/pull/515

File: ci/scenarios/edge/reroute/main.py [hunk 1]
```diff
@@ -224,7 +223,7 @@ def test_reroute_fio(fio_bin_path, vpool, compute_client, cluster_info, api, dis
         for index in xrange(0, disk_amount):
             try:
                 vdisk_name = '{0}_vdisk{1}'.format(EdgeTester.TEST_NAME, str(index).zfill(4))
-                data_vdisk = VDiskHelper.get_vdisk_by_guid(VDiskSetup.create_vdisk(vdisk_name, vpool.name, EdgeTester.AMOUNT_TO_WRITE, std_2.storage_ip, api))
+                data_vdisk = VDiskHelper.get_vdisk_by_guid(VDiskSetup.create_vdisk(vdisk_name, vpool.name, EdgeTester.AMOUNT_TO_WRITE * 2, std_2.storage_ip, api))
```
Comment (pploegaert) [inline ci/scenarios/edge/reroute/main.py]: why not put the default to 2 * 1024 ** 3 ?

File: ci/scenarios/vPool/add_remove_alba_vpool_test/main.py [hunk 2]
```diff
@@ -148,13 +150,16 @@ def validate_add_extend_remove_vpool(timeout=ADD_EXTEND_REMOVE_VPOOL_TIMEOUT):
 
         for cfg_name, cfg in vpool_configs.iteritems():
             # Create vpool
+            block_cache_cfg = None
+            if SystemHelper.get_ovs_version(SSHClient(SystemHelper.get_local_storagerouter(), username='root')).lower() == 'ee':
```
Comment (pploegaert) [inline ci/scenarios/vPool/add_remove_alba_vpool_test/main.py]: why not move the SSHClient ...get_local_storagerouter... up so we get sth more readable like:
if SystemHelper.get_ovs_version() == 'ee' ?

File: ci/scenarios/vPool/add_remove_alba_vpool_test/main.py [hunk 3]
```diff
@@ -242,10 +247,18 @@ def _add_vpool(vpool_name, fragment_cache_cfg, api, storagerouter_ip, albabacken
...
         :param timeout: specify a timeout
         :type timeout: int
+        :param block_cache_cfg: details of a vpool its block cache
+        :type block_cache_cfg: dict
         :param preset_name: name of a existing preset
         :type preset_name: str
         :return:
         """
+        vpool_cfg = {}
+        if block_cache_cfg is not None:
```
Comment (pploegaert) [inline ci/scenarios/vPool/add_remove_alba_vpool_test/main.py]: shouldn't we do this check for fragment_cache_cfg also?

File: ci/scenarios/vPool/add_remove_alba_vpool_test/main.py (hunk 2 above)
Comment (JeffreyDevloo) [inline ci/scenarios/vPool/add_remove_alba_vpool_test/main.py]: get_ovs_version needs the sshclient param. But I don't need it further in the code so no need to store it in memory.

File: ci/scenarios/vPool/add_remove_alba_vpool_test/main.py (hunk 3 above)
Comment (JeffreyDevloo) [inline ci/scenarios/vPool/add_remove_alba_vpool_test/main.py]: Fragment cache is a mandatory param, not an optional one.

File: ci/scenarios/edge/reroute/main.py (hunk 1 above)
Comment (JeffreyDevloo) [inline ci/scenarios/edge/reroute/main.py]: Disk size should be higher than the amount of data to write to be able to preallocate the space for fio

File: ci/scenarios/vPool/add_remove_alba_vpool_test/main.py (hunk 2 above)
Comment (kvanhijf) [inline ci/scenarios/vPool/add_remove_alba_vpool_test/main.py]: Nothing wrong with some readability in code

**Pull request merged**